### DISCUSSION TITLE
 ERROR  ENOENT: no such file or directory

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [pnpm/pnpm#3701] [Q&A]
![Annotation 2021-08-24 023634](https://user-images.githubusercontent.com/44552983/130515953-bfa4129b-9128-489c-a41b-ff3e393ce8f5.png)

node: v16.7.0
npm: v7.21.0
pnpm: v6.14.3

hey @zkochan please help me

**Comment by 0liver** [comment]
I've started seeing a very similar error a few days ago:

![ENOENT-chmod-pnpm](https://user-images.githubusercontent.com/164360/175693858-5a56a6a2-afb7-4dc1-9ed3-b563bfc4e851.png)


This happens during the build of a solution which contains a few projects that call `pnpm install` as part of their build process. I had never seen this error before in the same situation.

Any hints?
